lib/cartopy/mpl/gridliner.py

```diff
@@ -271,6 +271,9 @@ def __init__(self, axes, crs, draw_labels=False, xlocator=None,
         #: for styling of the text labels.
         self.ylabel_style = {}
 
+        # bbox style for grid labels
+        self.labels_bbox_style = None
```

Comment:
What about making the default the old version, which you can then overwrite?
`self.labels_bbox_style = {'pad': 0, 'visible': False}`

This would force you to add in the 'visible'=True keyword as well though, but it seems cleaner than all of the `if not None`'s below.

Reply:
I do agree, it would remove the ternary IFs. I'll change it tomorrow with some dict tests.